ranger/data/scope.sh

```diff
@@ -172,6 +172,8 @@ handle_image() {
 
         ## Video
         # video/*)
+        #     # don't generate thumbnail for incomplete torrent video file
+        #     [[ "$FILE_EXTENSION_LOWER" = "part" ]] && return
```

Comment:
for `break` i am getting `break: only meaningful in a for, while, or until loop` but `return` works

Reply:
Shell's case statement uses the rather less clear `;;&` for what I meant with `break`. The points raised by markus-bauer are interesting though, let's see if we can get a better solution first?

Reply:
Yeah, i agree with it. Can we make ranger to invoke only one scope.sh at a time. That way we don't need this pull request.